app/src/main/res/values/strings.xml

```diff
@@ -29,6 +29,10 @@
     <string name="sign_in_to_add_favorites">Please sign in to add favorites</string>
     <string name="filter">Filter</string>
     <string name="close">Close</string>
+    <string name="delete_dialog_title">Delete Item</string>
```

Comment:
No...  Why did  you add them a second time ? They were already there with the good naming convention.